### DISCUSSION TITLE
ZOOM - Best practice...?

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#1310] [General]
Hi @mtdvlpr 

I found your reply really interesting so thought I would transfer to this thread to keep the other one uncluttered. 

> ...We have an organization account managed by our branch, so the waiting room is locked to be always on.

We have a private account so can turn off the waiting room so that the friends can use the ZOOM account for field service without needing all the login details. 

> We have a scheduled recurring meeting that we start each time. 

Due to the waiting room being dynamic for us, we have never been able to use the 'recurring' scheduled meeting as when we turn off the waiting room it changes for all future meetings... was my understanding... but if I am wrong then that is great as I would love to be able to simplify things...

> Our starting point is no audio and video off and allow participants to unmute. Participants can talk to each other without us hearing them.

That is a great idea! I had never thought of that. That nullifies the need for the ‘complexity’ (for some hosts) of a breakout room. 

> 1 minute before the meeting, music in the hall stops, everyone is muted and can't unmute themselves anymore. We connect to the audio and turn the mic and video on.

I like that workflow! 
The only trouble is that we are supposed to check that KH audio is going to ZOOM okay, but if the setup is identical every time then audio should “just work” every time the same. 
> We don't share computer audio…

How do you get the music / video sound to be as good as it should be? We find ZOOM is set up for voice but music audio is horrible unless we share PC audio throughout the meeting (since M<sup>3</sup> & OBS cares for the rest).  What is your secret? Again, I would dearly love to simplify our setup!
 
> …and only use spotlight for Zoom participants with a part. 

That makes sense. If there is only one unmuted mic then ZOOM will select that camera/video anyway I guess. 

> We count the participants manually by looking at the names/video.

We do both since not everyone “chooses” to participate in the poll. Many of our participants choose to not use their camera/video so we don’t know / cannot see who is on the other side, and the numbers in some households can regularly change.  We encourage them to include their count with their join name but most don’t or don’t know how (despite training). 

> After the meeting we disconnect the audio, turn video off and allow participants to unmute themselves again.

Again, that is a really interesting workflow. I must try “don’t join with audio” and “disconnect audio” next time I am host.  Sometimes our Audio operator on the mixer forgets to turn down the ZOOM mic so we get to hear a few seconds of greetings from the ZOOM participants to each other, but if there is no audio then we won’t have that issue. 😁

**Comment by mtdvlpr** [comment]
Hi @Chipperchap, hope this helps! 

> Due to the waiting room being dynamic for us, we have never been able to use the 'recurring' scheduled meeting as when we turn off the waiting room it changes for all future meetings... was my understanding... but if I am wrong then that is great as I would love to be able to simplify things...

Correct me if I'm wrong, but if you schedule the meeting with waiting room off, you can turn it on during the meeting through the security menu. That way by default there's no waiting room, but you can turn it on when necessary.

> How do you get the music / video sound to be as good as it should be? We find ZOOM is set up for voice but music audio is horrible unless we share PC audio throughout the meeting (since M3 & OBS cares for the rest). What is your secret? Again, I would dearly love to simplify our setup!

From what I've experienced turning on 'original sound for musicians' did the trick (it must be enabled in settings once and then in every meeting it needs to be toggled on). This option prevents Zoom from trying to optimize the audio itself and lets the audio setup at the KH do the work. Also in OBS everything related to sound is turned off to prevent OBS from messing with it. We've only recently had the issue that the sound in Zoom is too loud, but I think that's resolved by setting the Zoom mic sensitivity manually instead of automatic. That works for us, but it might be that a different audio setup results in a different outcome.

**Comment by Chipperchap** [comment]
> Correct me if I'm wrong, but if you schedule the meeting with waiting room off, you can turn it on during the meeting through the security menu. That way by default there's no waiting room, but you can turn it on when necessary.

Correct - or at least that is also how I understand it. However that method does not allow the intro welcome video to be available (which is of course no big deal - just looks pretty that's all).  I suppose there are as many steps involved in one method to that of the other for us since we use the POLL feature and need to login to ZOOM via a browser to download the POLL results. 😁

> From what I've experienced turning on 'original sound for musicians' did the trick (it must be enabled in settings once and then in every meeting it needs to be toggled on). 

There is something about our setup that prevents this as an option. We did try it for a time but got complaints.  I have never figured out why as we have sound coming direct from the mixer to the PC as a "Rec out" via a converter from analogue to digital (USB), so the signal should be pure. The "share PC audio" is the only way we get pristine sound regardless of voice or music.

Thank you for your tips though! Keep 'em coming if you or others have any more! 😊

**Comment by mtdvlpr** [comment]
My pleasure :)
